src/alert.h

```diff
@@ -102,7 +102,7 @@ class CAlert : public CUnsignedAlert
     bool RelayTo(CNode* pnode) const;
     bool CheckSignature(const std::vector<unsigned char>& alertKey) const;
     bool ProcessAlert(const std::vector<unsigned char>& alertKey, bool fThread = true); // fThread means run -alertnotify in a free-running thread
-    static void Notify(const std::string& strMessage, bool fThread);
+    static void NotifyInternal(const std::string& strMessage);
```

Comment:
Wouldn't "NotifyExternal" be a more appropriate name that it is exposed while the static function is only used internally?